### PR TITLE
Rename SetupCode to AccountKey in translation keys

### PR DESCRIPTION
**Pull request by SailReal**
As discussed, user-facing stuff, including translation keys, is now called "Account Key". Everywhere else in the code it is called "Setup Code".

### CODE REVIEW

File: frontend/src/components/InitialSetup.vue [hunk 1]
```diff
@@ -88,7 +88,7 @@
                   <input id="confirmSetupKey" v-model="confirmSetupKey" name="confirmSetupKey" type="checkbox" class="h-4 w-4 rounded border-gray-300 text-primary focus:ring-primary" required>
```
Comment (tobihagemann) [inline frontend/src/components/InitialSetup.vue:88]: I noticed a "Setup Key" here. :smile:

Comment (SailReal) [reply frontend/src/components/InitialSetup.vue]: Good catch. Impressive how many variations of this term we had.

**Review by overheadhunter — CHANGES_REQUESTED** [review]
Isn't it confusing if the translation *keys* don't match what they technically belong to?

I'd rather keep it as it (well rename setupKey to setupCode is fine of course)

**Comment by tobihagemann** [comment]
It's confusing either way. I prefer not to confuse translators because they actually use the key as context. We have already received reports that the key looks wrong.

But since it's confusing either way, I'm also fine with the other option.

**Review by overheadhunter — APPROVED** [review]

**Pull request merged**